test/pr/terraform/modules/common/resource_policy/output/resources.json

```diff
@@ -0,0 +1,21 @@
+{
+  "planned_values": {
+    "root_module": {
+      "resources": [
+         {
+            "address": "google_compute_resource_policy.placement_policy",
+            "values": {
+              "group_placement_policy": [
+                {
+                  "availability_domain_count": 1,
+                  "collocation": "COLLOCATED",
+                  "vm_count": null
```

Comment:
I think we need to specify `max_distance = 2`

Reply:
added max_distance